core/modules/user/user.module

```diff
@@ -746,12 +760,12 @@ function user_user_view($account) {
 }
 
 /**
- * Helper function to add default user account fields to user registration and edit form.
+ * Helper function to add default user account fields to user registration and
+ * edit form.
  *
  * @see user_account_form_validate()
  * @see user_validate_current_pass()
  * @see user_validate_picture()
- * @see user_validate_mail()
```

Comment:
Since the email address is validated with `valid_email_address()`, should not this line be replaced by `@see valid_email_address()`?